jiraannouncer/views/fcms.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+from pyramid.view import view_config
+
+from ..utils import send, devsay, from_hex
+import logging
+
+log = logging.getLogger(__name__)
+
+
+@view_config(route_name='fcms', renderer="json")
+def fcms(request):
+    try:
+        data = request.json_body
+    except:
+        log.error("Failed to decode JSON from body. Dump:")
+        log.debug(request.body)
+        devsay("A FCMS payload couldn't be decoded. NoLifeKing, check the logfile!",
+               request)
+        return
+
+    carrier = data['carrier_callsign']
+    carrier_name = from_hex(data['carrier_vanity_name'])
+    target_system = data['calendar_arrivalSystem']
+    departure_system = data['current_starsystem']
+    jump_time = data['calendar_end'] if 'calendar_end' in data else None
```

Comment:
This is not an automated calendar event, it is one added by the player. If anything, they'll be setting their jump time as calendar_start and omitting calendar_end. Calendar_start is a required field, calendar_end is not.

Reply:
Oh, I thought this was scheduled through the game. I haven't played anything with carriers.

So start is when the jump should start?